### PR TITLE
[KP-712] Add secret key files handling

### PR DESCRIPTION
**Pull request by graddaniel**
https://kaholo.atlassian.net/browse/KP-712

### CODE REVIEW

File: helpers.js [hunk 1]
```diff
@@ -16,11 +23,37 @@ function readActionArguments(action, settings) {
       paramValues[paramDefinition.name],
       settingsValues[paramDefinition.name],
     );
+
+    validateParamValue(
```
Comment (xkcm) [inline helpers.js:27]: What if a parameter doesn't have `validationType` field specified?

File: parsers.js [hunk 2]
```diff
@@ -19,6 +19,8 @@ function resolveParser(type) {
       return autocomplete;
     case "array":
       return array;
+    case "sshKey":
+      return sshKey;
```
Comment (xkcm) [inline parsers.js:23]: Why not just return `string` parser here? `sshKey` parser function is not exported anyway, so it is just a wrapper for `string` parser, right? Unless the `sshKey` parser will be more complex in the future and right now this function is just a basis for future development.

**Review by xkcm — CHANGES_REQUESTED** [review]

File: helpers.js (hunk 1 above)
Comment (graddaniel) [reply helpers.js:27]: Good catch

File: parsers.js (hunk 2 above)
Comment (graddaniel) [reply parsers.js:23]: Initially the plan was to code it, but then I've realised I could just reuse string.
Good one.

**Review by xkcm — DISMISSED** [review]

**Pull request merged**